src/pkg/services/m365/api/lists.go

```diff
@@ -490,20 +518,57 @@ func setAdditionalDataByColumnNames(
 	fieldData := orig.GetAdditionalData()
 	filteredData := make(map[string]any)
 
-	for colName, colDetails := range columnNames {
-		if val, ok := fieldData[colName]; ok {
+	for _, colDetails := range columnNames {
+		if val, ok := fieldData[colDetails.getFieldName]; ok {
 			setMultipleEnabledByFieldData(val, colDetails)
-
-			filteredData[colName] = fieldData[colName]
+			filteredData[colDetails.createFieldName] = val
+			populateMultipleValues(val, filteredData, colDetails)
 		}
 
-		specifyODataType(filteredData, colDetails, colName)
+		specifyODataType(filteredData, colDetails, colDetails.createFieldName)
 	}
 
 	return filteredData
 }
 
+func populateMultipleValues(val any, filteredData map[string]any, colDetails *columnDetails) {
+	if !colDetails.isMultipleEnabled {
+		return
+	}
+
+	if !colDetails.isPersonColumn {
+		return
+	}
+
+	multiNestedFields, ok := val.([]any)
+	if !ok || len(multiNestedFields) == 0 {
+		return
+	}
+
+	lookupIDs := make([]float64, 0)
+
+	for _, nestedFields := range multiNestedFields {
+		if md, ok := nestedFields.(map[string]any); ok && keys.HasKeys(md,
+			[]string{LookupIDKey, LookupValueKey, PersonEmailKey}...) {
+			lookupID, ok := md[LookupIDKey].(*float64)
```

Comment:
this transform to a float64 should use the common/tform package (github won't let me suggest only line 553, so just putting the example here).
```suggestion
	v, err := tform.AnyValueToT[float64](lookupIDKey, md)
```